test/WebSites/ApiExplorerWebSite/Controllers/ApiExplorerResponseTypeWithoutAttributeController.cs

```diff
@@ -10,6 +10,7 @@ namespace ApiExplorerWebSite
     public class ApiExplorerResponseTypeWithoutAttributeController : Controller
     {
         [HttpGet]
+        [ProducesResponseType(typeof(void), 204)]
```

Comment:
Yes that's right. Per earlier comment, we would the users to explicitly set this.